app/routes/wow.ilvl-export-search.tsx

```
@@ -332,12 +352,16 @@ const IlvlExportSearchComponent = () => {
             { label: 'Realm Ranking', value: 'realmRanking' }
           ]}
-          onChange={(e) => setFormValues(prev => ({ ...prev, sortBy: e.target.value }))}
+          onChange={(e) =>
+            setFormValues((prev) => ({ ...prev, sortBy: e.target.value }))
+          }
         />
         <div className="flex flex-col gap-2">
-          <label className="text-sm font-medium dark:text-gray-200">Desired Stats</label>
+          <label className="text-sm font-medium dark:text-gray-200">
+            Desired Stats
+          </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)